api/scpca_portal/test/management/commands/test_load_data.py

```diff
@@ -54,8 +59,29 @@ def assertProjectData(self, project):
         self.assertIsNotNone(sample.tissue_location)
         self.assertIsNotNone(sample.treatment)
 
-    def assertProjectReadmeContains(self, text, project_zip):
-        self.assertIn(text, project_zip.read("README.md").decode("utf-8"))
+    def assertProjectReadmeContent(self, zip_file):
+        def get_masked_content(content):
+            """
+            Replace the project ID and date with placeholders for format testing.
+            """
+            content = re.sub(r"SCPCP\d{6}", "SCPCPXXXXXX", content)
+            content = re.sub(
+                r"Generated on: \d{4}-\d{2}-\d{2}", "Generated on: YYYY-MM-DD", content
+            )
+
+            return content
+
+        if README_FILE in zip_file.namelist():
```

Comment:
Let's remove this check, because we should always expect a readme here.

Reply:
See at 09eed514808eb4537f50e1d6da23c34045835049.